src/ResourceManager/Insights/Commands.Insights.Test/ScenarioTests/ActivityLogsTests.ps1

```diff
@@ -120,7 +120,71 @@ function Test-GetAzureSubscriptionIdLog
         $actual = Get-AzureRmLog -starttime 2015-01-15T12:30:00Z -endtime 2015-01-15T20:30:00Z 
 
         # Assert
-        Assert-AreEqual $actual.Count 1
+        Assert-AreEqual 1 $actual.Count
+    }
+    finally
+    {
+        # Cleanup
+        # No cleanup needed for now
+    }
+}
+
+<#
+.SYNOPSIS
+Tests getting the logs for a subscription Id.
+#>
+function Test-GetAzureSubscriptionIdLogMaxEvents
```

Comment:
@gucalder are these the tests containing the pages of size 10 that return 15 elements? (the test you mentioned in the email)